### ISSUE TITLE
Can we include the VSNU list in this repo?

### ISSUE THREAD

**Issue by bvreede** [bvreede/OAmonitoring#10]
It would be helpful to include essential data, namely the VSNU list of publications. Not sure if this is possible though so... @boer0107 ? What do you think?

**Comment by boer0107** [comment]
You mean open I assume? I will have to check with Arjan Schalken.

**Comment by bvreede** [comment]
Yes, I mean open! It is instrumental data for the classification, so it would be good to be able to include it. That said, we can write instructions on how to obtain it in the readme and be done with it.

**Comment by bvreede** [comment]
(This repo is open so by default everything included in it is open too :))
